src/shared/version_op.c

```diff
@@ -423,13 +424,23 @@ os_info *get_unix_version()
                         info->os_name = strdup(name);
                     }
                 } else if (strcmp (tag,"VERSION") == 0) {
-                    if (!version) {
+                    if (!version) {    
+                        if (version_id)
```

Comment:
Coding style: it might be safer to add curly braces to the **if** block.

Reply:
Ok, I will change it